packages/js/src/analysis/getWritingDirection.js

```diff
@@ -0,0 +1,16 @@
+import getL10nObject from "./getL10nObject";
+
+/**
+ * Gets the writing direction of the current page, either term or post.
+ *
+ * @returns {string} The writing direction of the page.
+ */
+export default function() {
+	let writingDirection = "LTR";
+
+	if ( getL10nObject().isRtl ) {
+		writingDirection = "RTL";
+	}
+
+	return writingDirection;
+}
```

Comment:
I created a separate helper file for this so that in the future it's easier for us to add more writing direction